### PR TITLE
Add template.json for ruleset-example

### LINKED ISSUE #15: ruleset-example is missing template config
Unlike the other templates, `ruleset-example` does not have a `.template.config` folder, meaning it will not be usable in `dotnet new`.

### PR DESCRIPTION
**Pull request by holly-hacker**
Adds a template for ruleset-example. Untested. The `identity` field is the same as for the ruleset-scrolling-example template, I don't know if that's an issue.

Closes #15

### CODE REVIEW

**Review by frenzibyte — DISMISSED** [review]

**Comment by frenzibyte** [comment]
I think remaming the scrolling ruleset example to `ScrollingPippidon` (or at least in the identity) would be better in my opinion

**Comment by holly-hacker** [comment]
Looked at the [documentation](https://docs.microsoft.com/en-us/dotnet/core/tools/custom-templates#templatejson) for template.json earlier and it seems the identity field has to be unique.

**Comment by holly-hacker** [comment]
Appended .Scrolling to the scolling example's identity, so it is in line with the other naming. Tested by generating a project using each template and they all build correctly.

File: templates/ruleset-scrolling-example/.template.config/template.json [hunk 1]
```diff
@@ -5,12 +5,12 @@
         "Console"
     ],
     "name": "osu! ruleset (scrolling pippidon example)",
-    "identity": "ppy.osu.Game.Templates.Rulesets.Pippidon",
+    "identity": "ppy.osu.Game.Templates.Rulesets.Pippidon.Scrolling",
```
Comment (frenzibyte) [inline templates/ruleset-scrolling-example/.template.config/template.json]: why not `Rulesets.Scrolling.Xxx` as an identity for any scrolling ruleset?
```suggestion
    "identity": "ppy.osu.Game.Templates.Rulesets.Scrolling.Pippidon",
```

**Review by frenzibyte — CHANGES_REQUESTED** [review]

File: templates/ruleset-scrolling-example/.template.config/template.json (hunk 1 above)
Comment (holly-hacker) [reply templates/ruleset-scrolling-example/.template.config/template.json]: Fine by me, but it's not like this is used for anything or is shown to anyone.

**Review by frenzibyte — APPROVED** [review]

**Review by peppy — APPROVED** [review]

**Pull request merged**